src/common/ers.c

```diff
@@ -220,7 +220,7 @@ static void *ers_obj_alloc_entry(ERS *self)
 		instance->Cache->ReuseList = instance->Cache->ReuseList->Next;
 	} else if (instance->Cache->Free > 0) {
 		instance->Cache->Free--;
-		ret = &instance->Cache->Blocks[instance->Cache->Used - 1][instance->Cache->Free * instance->Cache->ObjectSize + sizeof(struct ers_list)];
+		ret = &instance->Cache->Blocks[instance->Cache->Used - 1][instance->Cache->Free * (size_t)instance->Cache->ObjectSize + (int)sizeof(struct ers_list)];
```

Comment:
in this line cast for ``(int)`` exists but in line 234 is not. this is correct?

Reply:
Oops, no. My bad, thank you for pointing it out